Tests/PetaPoco/PetaPoco.cs

```diff
@@ -283,13 +283,13 @@ static void AddParam(DbCommand cmd, object item, string ParameterPrefix)
 			}
 			else
 			{
-				if (item.GetType() == typeof(Guid))

```

Comment:
PetaPoco is a separate (similar) tool used in the tests for comparison purposes; we should not be editing this code (unless to update to a new version)